client/java-armeria/src/main/java/com/linecorp/centraldogma/client/armeria/ArmeriaCentralDogma.java

```diff
@@ -150,8 +151,10 @@
     }
 
     private static Void createProject(AggregatedHttpMessage res) {
-        if (res.status().code() == 201) { // Created
-            return null;
+        switch (res.status().code()) {
+            case 200:
```

Comment:
Question: [ProjectServiceV1.createProject()](https://github.com/line/centraldogma/blob/master/server/src/main/java/com/linecorp/centraldogma/server/internal/api/ProjectServiceV1.java#L92) always returns `201` when it succeeded to create a new project. So would it be better to handle this as an error like before? 🤔 
(It's the same question for the removal APIs)

Reply:
Just wanted to relax the response status code check so that the client doesn't have to change when we make a server-side change for some reason. Shall I revert this?

Reply:
No. :-) Thank you for explanation.